impeller/renderer/renderer_unittests.cc

```diff
@@ -514,7 +514,7 @@ TEST_P(RendererTest, CanBlitTextureToTexture) {
       // Blit `bridge` to the top left corner of the texture.
       pass->AddCopy(bridge, texture);
 
-      pass->EncodeCommands(context->GetResourceAllocator());
+      assert(pass->EncodeCommands(context->GetResourceAllocator()));
```

Comment:
`assert`s get stripped away in !defined(NDEBUG). Besides, let's not terminate the process on this error. Return false like elsewhere?

Reply:
Whoops, fixed.